runtime/runtime_events.c

```diff
@@ -103,7 +103,7 @@ static HANDLE ring_handle;
 
 /* This comes from OCAMLRUNPARAMs and is initialised in
    caml_runtime_events_init */
-static int ring_size_words;
+int caml_runtime_events_ring_size_words;
```

Comment:
I'm not clear why this needs exposing? Should `caml_runtime_parameters` simply be displaying `caml_params->runtime_events_log_wsize`?